### ISSUE TITLE
different icons for .gltf and .tscn

### ISSUE THREAD

**Issue by tc4r37a7b** [godotengine/godot-proposals#3043]
![image](https://user-images.githubusercontent.com/87856877/126910866-4814226c-73b4-4dc6-84e3-bb30106a4fa6.png)

something like this for all different types of 3D files?

image reference, https://thenounproject.com/term/gltf/

**Comment by Calinou** [comment]
In the interest of making high-quality proposals, please [read the rules](https://github.com/godotengine/godot-proposals#readme) and fill in the issue template. This proposal will be reopened once this is done :slightly_smiling_face:

**Comment by tc4r37a7b** [comment]
I do not know how a simple request like that can fit in such amount of redundant questions (in this specific case).

**Comment by pycbouh** [comment]
There is no request too small when it comes to properly describing your idea 🙃 You could still show how the current state is a problem and how you propose to improve it. Especially for those of us who don't have the context you have.

Alternatively, a direct PR is welcome for something simple.

**Comment by tc4r37a7b** [comment]
where I do left a PR message?

**Comment by pycbouh** [comment]
> where I do left a PR message?

I'm sorry, can you rephrase that?

**Comment by tc4r37a7b** [comment]
You typed "Alternatively, a direct PR is welcome for something simple."
As not understanding what PR means. I asked where I should leave this suggestion.

**Comment by pycbouh** [comment]
PR stands for "pull request". It's the way you contribute code to an open source project, by creating a commit/a patch in your copy of the repository and then creating a pull request, literally a request to pull your changes into the main repository.

What I meant by the last sentence is that you can fix it and submit the fix if it's a simple and not controversial change, instead of making a proposal for it.
